### ISSUE TITLE
Improvement for node close to metric

### ISSUE THREAD

**Issue by HugoJourdan** [mekkablue/ShowAngledHandles#22]
The latest indicator added, which shows a point one unit close to a metric, is currently working for all metrics. 
Perhaps it could be useful to skip some metrics based on the case of the layer?

```
If case == 1 : #Uppercase
    ignore xHeight 

if case == 2 : #Lowercase
    ignore CapHeight
```

**Comment by mekkablue** [comment]
Good idea. There are some tricky edge cases, though. I’ll see what I can do.
